x-pack/dockerlogbeat/docs/troubleshooting.asciidoc

```diff
@@ -0,0 +1,7 @@
+[[log-driver-troubleshooting]]
+== Troubleshooting
+
+experimental[]
+
+
+TODO: Move content about troubleshooting from readme to this section.
```

Comment:
Maybe put logging info here?

Reply:
@fearful-symmetry So it sounds like the only option for logging is to set the log driver level in the env var?

Reply:
Yep, unfortunately. It's a limitation of libbeat due to how the plugin uses libbeat.